src/Core/Style.js

```diff
@@ -26,6 +26,7 @@ class Style {
      * @param {object} [params]
      * @param {object} [params.fill] fill style.
      * @param {string} [params.fill.color] fill color string css.
+     * @param {Image|Canvas} [params.fill.pattern] fill with pattern image.
```

Comment:
I would prefer allowing a `THREE.Texture`, then you can test it using `isTexture`.

Reply:
Canvas context couldn't use `THREE.Texture`.

Reply:
Yes, but below you can then use `params.fill.pattern.image`. This way the texture could be updated without having to change the style ?